src/tool/cppxlang/type_writers.h

```diff
@@ -209,40 +209,20 @@ namespace xlang
 
             // TODO: get rid of all these renames once parity with cppxlang.exe has been reached...
 
-            if (name == "EventRegistrationToken" && ns == "Windows.Foundation")
+            if (name == "EventRegistrationToken" && ns == "Foundation")
```

Comment:
For language projections that build on top of C++ projection, it's a pain in the ass to undo the customization of these core structs. Is there any way to add a command line option (perhaps in a later PR) to emit a C++ proecjtion w/o them?

Reply:
It goes further than that. Now that I've laid down the initial cppx language projection (and your Python winrt work), we've got a much better idea of where the common code is, and where the customization points need to be. Let's make sure we have a task to perform some consolidation work into a core projection helper library. That consolidation work is the perfect time to account for these behavior differences.